modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImplTest.java

```diff
@@ -462,48 +464,6 @@ void testErrorOnCursorInitialization() throws InterruptedException {
         awaitContextCancellation(execNodes);
     }
 
-    /**
-     * The very simple case where a query is cancelled in the middle of a normal execution on non-initiator node.
-     */
-    @Test
-    public void testCancelOnRemote() throws InterruptedException {
```

Comment:
Just curious. Were we really able to cancel a request from any node?

Reply:
technically, yep. We were cancelled any fragment, and the QueryCancellation error was propagated to head of the query, and then back to remaining fragments.

but it never was in use, and now we have another mechanism for cancellation